scripts/scripts.js

```diff
@@ -249,3 +249,57 @@ async function loadPage() {
+    return `${value} ${value === 1 ? 'month' : 'months'} ago`;
+  }
+  interval = seconds / 86400;
+  if (interval > 1) {
+    const value = Math.floor(interval);
+    return `${value} ${value === 1 ? 'day' : 'days'} ago`;
+  }
+  interval = seconds / 3600;
+  if (interval > 1) {
+    const value = Math.floor(interval);
+    return `${value} ${value === 1 ? 'hour' : 'hours'} ago`;
+  }
+  interval = seconds / 60;
+  if (interval > 1) {
+    const value = Math.floor(interval);
+    return `${value} ${value === 1 ? 'minute' : 'minutes'} ago`;
+  }
+  const value = Math.floor(interval);
+  return `${value} ${value === 1 ? 'second' : 'seconds'} ago`;
+};
+
+/**
+ * Converts excel date into JS date.
+ * @param {number} excel date to convert.
+ */
+export const convertExcelDate = (excelDate) => {
+  const secondsInDay = 86400;
+  const excelEpoch = new Date(1899, 11, 31);
+  const excelEpochAsUnixTimestamp = excelEpoch.getTime();
+  const missingLeapYearDay = secondsInDay * 1000;
+  const delta = excelEpochAsUnixTimestamp - missingLeapYearDay;
+  const excelTimestampAsUnixTimestamp = excelDate * secondsInDay * 1000;
+  const parsed = excelTimestampAsUnixTimestamp + delta;
+  return Number.isNaN(parsed) ? null : new Date(parsed);
+};
+
+export const gdPlusIcon = 'data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNTEiIGhlaWdodD0iMTkiIHZpZXdCb3g9IjAgMCA1MSAxOSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiByb2xlPSJpbWciIGZpbGw9IiMwMDAiIGFyaWEtbGFiZWw9IkdEIFBsdXMgTG9nbyIgYXJpYS1oaWRkZW49InRydWUiPjxwYXRoIGNsYXNzPSJnZC10ZXh0IiBkPSJNMTIuMjMxIDE3LjY2MjVDMTEuODk5NCAxNy44MjgyIDExLjI2NDQgMTguMDIxMSAxMC43Njc2IDE4LjA3NjNWMTguNDY0QzE0LjA4MTIgMTguMzgwNiAxNi40MDA3IDE3LjgyOTIgMTcuODkzMiAxNy4xNjYzVjEwLjU5NTJIMTIuMjMxVjE3LjY2MjVaIiA+PC9wYXRoPjxwYXRoIGNsYXNzPSJnZC10ZXh0IiBkPSJNMTcuNDUwNSA0Ljg1MzExVjEuMTU0NDdDMTYuMjYzOSAwLjgyMzAzIDE0LjMzMDcgMC42MDI1NiAxMi4yODcxIDAuNTczVjAuOTYwNjVDMTQuMzg1NSAxLjk1NTQ1IDE1Ljg3NjYgMy4wODY4OSAxNy40NTA1IDQuODUzMTFaIiA+PC9wYXRoPjxwYXRoIGNsYXNzPSJnZC10ZXh0IiBkPSJNNy4zNzEzOCA5LjUxNzVDNy4zNzEzOCA0LjMwMDIzIDguOTQ1MzEgMS41MzkyMSAxMS4yMSAwLjk2MDE2VjAuNTcyNTFDNS4wMjQyOSAwLjk4NjMyIDAuOTM3NSA0LjI0NDAyIDAuOTM3NSA5LjUxNjVDMC45Mzc1IDE0LjY1MjggNC41NTQ1OSAxOC4xODM4IDEwLjAyMjQgMTguNDYwNVYxOC4wNzI4QzguMDM0NDkgMTcuNzE2MiA3LjM3MTM4IDE0LjUxMzggNy4zNzEzOCA5LjUxNzVaIiA+PC9wYXRoPjxwYXRoIGNsYXNzPSJnZC10ZXh0IiBkPSJNMjcuNDE4NyAwLjgyM0gyMS4xNzc3VjE4LjIxNDNIMjcuNDE4N1YwLjgyM1oiID48L3BhdGg+PHBhdGggY2xhc3M9ImdkLXRleHQiIGQ9Ik0yOC4zODQ4IDAuODIzVjEuMjM3NzhDMzAuODQyOSAxLjQ1ODI1IDMyLjAwMTkgMy4xNjk3MiAzMi4wMDE5IDkuNTE4OUMzMi4wMDE5IDE1LjU5MTkgMzAuNzkgMTcuNjMzOCAyOC40MTQ4IDE3LjgyNzZWMTguMjE1M0MzNC42Mjc3IDE4LjEzMjQgMzguNDM4NiAxNS4yMDYyIDM4LjQzODYgOS41MTk5QzM4LjQzNjcgMy4zMDc4MiAzNC40ODc2IDAuOTA2MzQgMjguMzg0OCAwLjgyM1oiID48L3BhdGg+PHBhdGggZD0iTTQ2LjY4NzUgNS4yMjQ4NUg0NC44MTI1VjEzLjg0OTlINDYuNjg3NVY1LjIyNDg1WiIgZmlsbD0iI0VEM0U0OSI+PC9wYXRoPjxwYXRoIGQ9Ik00MS40Mzc1IDguNTk5OVYxMC40NzQ5SDUwLjA2MjVWOC41OTk5SDQxLjQzNzVaIiBmaWxsPSIjRUQzRTQ5Ij48L3BhdGg+PC9zdmc+'; // prettier-ignore
```

Comment:
You can replace this inline SVG with a reference to an image SVG from the icons folder eg `/icons/gdplus.svg`. The scripts.js is loaded early so it's not needed to load this SVG early on.